### ISSUE TITLE
8.4 installation issues

### ISSUE THREAD

**Issue by pierky** [openbgpd-portable/openbgpd-portable#75]
Hello,

I'm getting some issues while installing 8.4.
It seems that something changed upstream in openbgpd and autogen fails to apply a patch.

```
Copying http.h to bgplgd
Copying slowcgi.h to bgplgd
Patching patches/0001-depend-on-carp-requieres-HAVE_CARP.patch
Patching patches/0002-set-IPV6_V6ONLY-if-we-are-binding-to-an-IPv6-address.patch
Patching patches/0003-Special-handle-IPV6_MINHOPCOUNT-and-IP_MINTTL-since-.patch
Patching patches/0004-Allow-overriding-default-user-and-file-locations.patch
1 out of 2 hunks FAILED -- saving rejects to file bgpd/bgpd.h.rej
```

```
root@7f1aa4ad24c7:~/openbgpd-portable-8.4# cat ./src/bgpd/bgpd.h.rej
--- bgpd/bgpd.h
+++ bgpd/bgpd.h
@@ -35,8 +35,6 @@
 #define	BGP_VERSION			4
 #define	BGP_PORT			179
 #define	RTR_PORT			323
-#define	CONFFILE			"/etc/bgpd.conf"
-#define	BGPD_USER			"_bgpd"
 #define	PEER_DESCR_LEN			32
 #define	REASON_LEN			256	/* includes NUL terminator */
 #define	PFTABLE_LEN			32
```

Forcing the script to use 87d69541b23938c66121f7151c11796afe8fcc52 works.

**Comment by robert-scheck** [comment]
Confirming, the nightly GitHub Action for the container image [fails](https://github.com/openbgpd-portable/openbgpd-container/actions/runs/8353372689/job/22864989907#step:8:720) since today the same way.

**Comment by cjeker** [comment]
Fixed in 42da261565ae849fdaabf9813362d37f00446bbf

Just to be clear this is a problem with `-current`. The 8.4 version is not affected since releases don't need to run `./autogen.sh` the files in the official releases have these patches already applied.
